re_data/command_line.py

```diff
@@ -331,8 +338,8 @@ def generate(start_date, end_date, interval, re_data_target_dir, **kwargs):
     if dbt_vars: dbt_docs.extend(['--vars', yaml.dump(dbt_vars)])
     add_dbt_flags(dbt_docs, kwargs)
     dbt_docs_process = subprocess.run(dbt_docs)
-    dbt_docs_process.check_returncode()
-
+    if force is True:
```

Comment:
@yu-iskw looking at it, shouldn't it be?
`if force is not True:`

Reply:
That's true. I will fix it.